src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int randomIndex = new Random().nextInt(Color.values().length);
```

Comment:
Creating a new instance of 'Random' every time the method is called can be inefficient. It's better to have a single instance as a class member and reuse it.